### PR TITLE
[fix] fix fa unrecognized v2 events

### PR DESCRIPTION
**Pull request by lightmark**
fix fa events tracking in indexer
image

### CODE REVIEW

File: rust/processor/src/db/common/models/coin_models/coin_utils.rs [hunk 1]
```diff
@@ -324,6 +342,12 @@ impl CoinEvent {
             "0x1::coin::DepositEvent" => {
                 serde_json::from_str(data).map(|inner| Some(CoinEvent::DepositCoinEvent(inner)))
             },
+            "0x1::coin::CoinWithdraw" => {
+                serde_json::from_str(data).map(|inner| Some(CoinEvent::WithdrawCoinEventV2(inner)))
+            },
+            "0x1::coin::CoinDeposit" => {
+                serde_json::from_str(data).map(|inner| Some(CoinEvent::DepositCoinEventV2(inner)))
+            },
```
Comment (rtso) [inline rust/processor/src/db/common/models/coin_models/coin_utils.rs:350]: why are we changing coin events? are they also affected?

Comment (lightmark) [reply rust/processor/src/db/common/models/coin_models/coin_utils.rs:350]: not really. I just checkout the file from the other PR. I can prune those.

**Comment by rtso** [comment]
How did you test this? Also do you know which versions we released FA v2 events? We need to backfill testnet and mainnet from those versions.

**Comment by lightmark** [comment]
image
@rtso 
vs https://cloud.hasura.io/public/graphiql?endpoint=https%3A%2F%2Findexer.mainnet.aptoslabs.com%2Fv1%2Fgraphql&query=query+MyQuery+%7B%0A++fungible_asset_activities%28where%3A+%7Btransaction_version%3A+%7B_eq%3A+%221022270660%22%7D%7D%29+%7B%0A++++amount%0A++++entry_function_id_str%0A++++gas_fee_payer_address%0A++++is_frozen%0A++++asset_type%0A++++owner_address%0A++%7D%0A%7D%0A

File: rust/processor/src/db/common/models/coin_models/coin_utils.rs [hunk 2]
```diff
@@ -151,6 +151,22 @@ pub struct DepositCoinEvent {
...
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct DepositCoinEventV2 {
+    pub coin_type: String,
+    pub account: String,
+    #[serde(deserialize_with = "deserialize_from_string")]
+    pub amount: BigDecimal,
+}
+
```
Comment (rtso) [inline rust/processor/src/db/common/models/coin_models/coin_utils.rs:169]: Remove this too

**Pull request merged**